roles/openshift-labels/test/test.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@InfoSec812 while these tests are good for what you are working on, or when paired up with the comment/documentation in this PR, they don't necessarily make sense to keep in the role - there's just too many dependencies needed in place for these to work. I'd suggest either adding to the playbook to create the necessary projects and objects (and, if so, please make it non labs-ci-cd specific), or just don't add these tests but rather a README at the top level of this role